lib/potentialbuild.rb

```diff
@@ -85,7 +85,9 @@ def initialize(client, token, repository, tag_name, commit_sha, branch_name, aut
     @asset_url = nil
     @acting_as_baseline = false
 
-    @performance_results = nil
+    @valgrind_counters_results = nil
+    @perf_counters_results = nil
+    @file_sizes = nil
```

Comment:
Keeping a few extra parameters on the potential build class, initially nil.  Fine.